File: src/EnfeiZhang_GrittyP3/Question6/FiftyToOneHundred.java
/*Enfei Zhang
  March 18, 2018
  Ms. Krasteva
  This is a program that will return a number between 50 and 100
 */

public class FiftyToOneHundred {
    public static void main(String[] args) {
        System.out.println((Math.random() * 51) + 50);  //Outputs a double
        System.out.println((int)(Math.random() * 51) + 50); //Outputs an integer
    }
}

File: Assignments/Weeklies/EnfeiZhang_GrittyP3/Question6/FiftyToOneHundred.kt
/*Enfei Zhang
  March 18, 2018
  Ms. Krasteva
  This is a program that will return a number between 50 and 100
 */

object FiftyToOneHundred {
    @JvmStatic
    fun main(args: Array<String>) {
        println(Math.random() * 51 + 50)  //Outputs a double
        println((Math.random() * 51).toInt() + 50) //Outputs an integer
    }
}
